example/src/app/app.component.html

```diff
@@ -248,38 +248,49 @@ <h4><b>based on angular2-color-picker by Alberto Pujante</b></h4>
 
   <div class="row">
     <div class="col-md-5">
-      <input [value]="color13" [style.background]="color13" [cpOutputFormat]="'rgba'" [cpAlphaChannel]="'disabled'" [(colorPicker)]="color13"/>
+      <input [value]="color13" [style.background]="color13" [cpAlphaChannel]="'disabled'" [cpOutputFormat]="'rgba'" [(colorPicker)]="color13"/>
 
       <br>
 
       <input [value]="color14" [style.background]="color14" [cpAlphaChannel]="'always'" [cpOutputFormat]="'rgba'" [(colorPicker)]="color14"/>
 
       <br>
 
-      <input [value]="color15" [style.background]="rgbaText" [cpAlphaChannel]="'always'" [cpOutputFormat]="'hex'" [colorPicker]="color15" (colorPickerChange)="rgbaText=onChangeColorHex8($event);color15=$event"/>
+      <input [value]="color15" [style.background]="color15" [cpAlphaChannel]="'forced'" [cpDefaultInput]="'hex'" [cpOutputFormat]="'rgba'" [(colorPicker)]="color15"/>
```

Comment:
This is the example with the new `cpAlphaChannel='forced'` and `cpDefaultInput` parameters